src/lib/drupal/facilityNoDrupalPageSideNav.ts

```diff
@@ -0,0 +1,23 @@
+import { SidebarData } from '@/types/formatted/staffProfile'
+
+export const buildSidebarData = (
```

Comment:
This seems like a not specific enough name? 😕 

Can you either give it a clearer name (there's already a function to build some shape of sidebar data floating around here) or at least document this to make it clear when we'd use this function instead of the other?

Reply:
Renamed along with everything related to `facilityNoDrupalPageSideNav`